FirebaseSwiftUI/FirebaseFacebookSwiftUI/Sources/Services/FacebookProviderSwift.swift

```diff
@@ -34,6 +35,10 @@ public class FacebookProviderSwift: FacebookProviderProtocol {
     shaNonce = CommonUtils.sha256Hash(of: rawNonce)
   }
 
+  @MainActor public var authButton: any View {
```

Comment:
I'm going to presume we should be avoiding "any View" as [mentioned in the main PR](https://github.com/firebase/FirebaseUI-iOS/pull/1237#discussion_r1983491150), unless it is unavoidable. Thoughts, @peterfriese?